crypto/x509/x509_vfy.c

```diff
@@ -118,33 +118,32 @@ static int null_callback(int ok, X509_STORE_CTX *e)
  * to match issuer and subject names (i.e., the cert being self-issued) and any
  * present authority key identifier to match the subject key identifier, etc.
  */
-static int x509_self_signed_ex(X509 *cert, int verify_signature,
-                               OPENSSL_CTX *libctx, const char *propq)
+static int x509_self_signed(X509 *cert, int verify_signature)
```

Comment:
I wonder why we need this as a separate function now. Can't this just be collapsed into `X509_self_signed()`